File: web-app/online-warehouse/src/app/counterparty/dto/counterparty.filter.ts
export class CounterpartyFilter {

  name: string;
  counterpartyType: string;
  taxNumber: string;


  constructor(name?: string, counterpartyType?: string, taxNumber?: string) {
    this.name = name;
    this.counterpartyType = counterpartyType;
    this.taxNumber = taxNumber;
  }
}

File: core/src/main/java/by/itechart/counterparty/dto/CounterpartyFilter.java
package by.itechart.counterparty.dto;

import by.itechart.counterparty.enums.CounterpartyType;
import lombok.Data;

@Data
public class CounterpartyFilter {

    private String name;
    private String taxNumber;
    private CounterpartyType counterpartyType;
}
